openapi.yaml

```diff
@@ -188,34 +188,42 @@ components:
           $ref: '#/components/schemas/Address'
       required:
         - first_name
-
+                  
     EmployeeRelationships:
       type: object
+      required: [jobs]
       properties:
-        location:
+        jobs:
           type: object
+          required: [data]
           properties:
             data:
-              type: object
-              properties:
-                id:
-                  type: string
-                  example: "1"
-                type:
-                  type: string
-                  example: "locations"
-        position:
-          type: object
-          properties:
-            data:
-              type: object
-              properties:
-                id:
-                  type: string
-                  example: "1"
-                type:
-                  type: string
-                  example: "positions"
+              type: array
+              minItems: 1
+              items:
+                type: object
+                required: [type, relationships]
+                properties:
+                  type:
+                    type: string
+                    enum: [jobs]
+                    example: jobs
+                  relationships:
+                    type: object
+                    required: [position, location]
```

Comment:
Within an object in the `jobs` array, there must be a `position` and a `location` specified.